client/src/utils/colors.tsx

```diff
@@ -0,0 +1,24 @@
+// TODO: map to color schemes
```

Comment:
Is this to-do to map each ingredient type to certain shades of color? ie. Red shades = meat?

Reply:
Just reviewed the new PR. All good now 👍